src/Rokt-Kit.js

```diff
@@ -214,11 +227,8 @@ var constructor = function () {
                         console.warn(
                             'Rokt Kit: No filtered user has been set.'
                         );
-                    } else {
-                        self.filteredUser = roktFilters.filteredUser;
                     }
```

Comment:
removing this line to avoid having 2 filtered users, 1 on `self.filteredUser`, and 1 on `self.filters.filteredUser` which is already set in line 212